### ISSUE TITLE
Ignore functionality on 3.x and contribution

### ISSUE THREAD

**Issue by parshav** [gojuno/mainframer#265]
Hey, so Im setting up the 3.x version of mainframer, and notice that the ignore.yml is not respected by mainframer. Looking further, I noticed that `Ignore::from_working_dir()` uses three different files, unlike what is specified in the Readme.
Although I can go in myself and modify the changes, is this the approach the tool will take henceforth ? Is there a board / scope of work for 3.x  that I could possibly help out with as well?
Thanks !

**Comment by artem-zinnatullin** [comment]
Yeah, sorry about that, we've updated some docs ahead of code. For now you [should define](https://github.com/buildfoundation/mainframer/blob/8454660cb6d0aa717b8915af42240376ac29e543/src/ignore.rs#L14) `.mainframer/ignore` file as it was in [2.x](https://github.com/buildfoundation/mainframer/tree/v2.1.0)

If you want to update the docs to reflect current state, I'd happily accept that change and then would update them again once we migrate all configs to YAML
